MekHQ/src/mekhq/campaign/finances/Money.java

```diff
@@ -163,6 +162,10 @@ public String toAmountAndNameString() {
         return CurrencyManager.getInstance().getUiAmountAndNamePrinter().print(this.wrapped.toMoney(RoundingMode.HALF_EVEN));
     }
 
+    public static Money parse(String moneyStr) {
+        return new Money(BigMoney.parse(moneyStr));
```

Comment:
The documentation for BigMoney#parse says it expects a String in the format of a three-letter currency code followed by a numeric value separated by one or more spaces. What happens when you don't have the code?
https://www.joda.org/joda-money/apidocs/org.joda.money/org/joda/money/BigMoney.html#parse(java.lang.String)

Reply:
It worked in testing, but I've decided to go another way with this that doesn't involve parsing strings.